setupext.py

```diff
@@ -531,6 +540,41 @@ def do_custom_build(self):
         """
         pass
 
+    def install_help_msg(self):
+        """
+        The help message to show if the package is not installed. The help
+        message shown depends on whether some class variables are present.
```

Comment:
Can you add a note to this docstring explaining how to add the feature to sub-classes?